mne/utils/tests/test_numerics.py

```diff
@@ -171,8 +171,38 @@ def test_time_mask():
     assert _time_mask(x, tmin=4.4999, sfreq=1).sum() == 2
     assert _time_mask(x, tmin=4, sfreq=1).sum() == 2
     # degenerate cases
-    pytest.raises(ValueError, _time_mask, x[:1], tmin=11, tmax=12)
-    pytest.raises(ValueError, _time_mask, x[:1], tmin=10, sfreq=1)
+    with pytest.raises(ValueError, match='No samples remain'):
```

Comment:
should be `match='No frequencies remain'` now :)

Reply:
You are looking at ```test_time_mask``` right there, so it's alright. It's already changed to ```match='No frequencies remain'``` in ```test_frequency_mask``` (else the tests wouldn't have passed).

Reply:
Ok, sorry, my bad. :)